pySDC/implementations/problem_classes/AllenCahn_2D_FD.py

```diff
@@ -1,11 +1,14 @@
+import time
```

Comment:
I don't see a reason why you need to touch this file in the PR. I understand that you added timings, but this should not be part of the PR.